troposphere/template_generator.py

```diff
@@ -307,10 +308,11 @@ def _get_function_type(self, function_name):
 
     def _import_all_troposphere_modules(self):
         """ Imports all troposphere modules and returns them """
+        dirname = os.path.join(os.path.dirname(__file__))
         module_names = [
             pkg_name
             for importer, pkg_name, is_pkg in
-            pkgutil.walk_packages(__file__)
+            pkgutil.walk_packages([dirname], prefix="troposphere.")
```

Comment:
Sorry, something seems to have gone wrong, I thought this had updated, but that does not seem to have happened.

Prefix needs to be kept, since we start by iterating over the troposphere package. Otherwise, the DEPRECATED_PACKAGES won't work as expected.

You are correct that pkg_name.startswith can be removed, because we have forced everything to start with troposphere.

I have uploaded this now.